### PR TITLE
Drop xvfb from jenkins master and workers

### PR DESCRIPTION
**Pull request by tbrisker**

### CODE REVIEW

File: puppet/modules/slave/manifests/init.pp [hunk 1]
```diff
@@ -303,15 +297,6 @@
     }
   }
 
-  # Cleanup Jenkins Xvfb processes from aborted builds after a day
-  file { '/etc/cron.daily/xvfb_cleaner':
-    ensure  => file,
```
Comment (ekohl) [inline puppet/modules/slave/manifests/init.pp]: Can you change this to `ensure => absent` so we actually remove it?

Comment (tbrisker) [reply puppet/modules/slave/manifests/init.pp]: updated

File: puppet/modules/slave/manifests/init.pp [hunk 2]
```diff
@@ -305,11 +299,7 @@
 
   # Cleanup Jenkins Xvfb processes from aborted builds after a day
   file { '/etc/cron.daily/xvfb_cleaner':
-    ensure  => file,
-    owner   => 'root',
-    group   => 'root',
-    mode    => '0755',
-    content => "#!/bin/sh\nps -eo pid,etime,comm | awk '(\$2 ~ /-/ && \$3 ~ /Xvfb/) { print \$1 }' | xargs kill >/dev/null 2>&1 || true\n",
+    ensure  => absent,
```
Comment (ekohl) [inline puppet/modules/slave/manifests/init.pp]:
```suggestion
    ensure => absent,
```

**Comment by ekohl** [comment]
I'll let the tests continue but otherwise good to merge.

**Comment by tbrisker** [comment]
tests are green, feel free to squash

**Pull request merged**